src/transformers/training_args.py

```diff
@@ -144,6 +146,8 @@ class TrainingArguments:
         warmup_steps (:obj:`int`, `optional`, defaults to 0):
             Number of steps used for a linear warmup from 0 to :obj:`learning_rate`. Overrides any effect of
             :obj:`warmup_ratio`.
+        log_level (:obj:`str`, `optional`):
+            Logging level - same as logging log levels.
```

Comment:
The help below contains more info that would be great here too! Plus you indicate the default in the arg description.

Reply:
replicated and fixed the default. Thank you!